application-accelerator/creating-accelerators/creating-accelerators.hbs.md

```diff
@@ -198,10 +201,12 @@ To avoid having to create a separate manifest file, you can use the following co
 ```
 tanzu accelerator fragment create java-version \
   --git-repo https://github.com/vmware-tanzu/application-accelerator-samples.git \
-  --git-tag tap-1.3 \
+  --git-tag ${TAP_VERSION} \
   --git-sub-path fragments/java-version
 ```
 
+Where `TAP_VERSION` is the `spec.git.ref.tag` value that is located in the `java-version.yaml`. (e.g. `tap-1.4`)
```

Comment:
This is confusing - the tag is the git repo tag used for specific versions of the samples that are delivered for each TAP release like `tap-1.3`, `tap-1.3.1` or `tap-1.4.0` - you can see the list here: https://github.com/vmware-tanzu/application-accelerator-samples/tags

Reply:
I'll rephrase this to call out the git tag rather than the TAP version. Good call 👍